service/orchestrator/toe_test.go

```diff
@@ -0,0 +1,362 @@
+	"clouditor.io/clouditor/persistence"
+	"clouditor.io/clouditor/persistence/gorm"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var AssuranceLevelHigh = "high"
+var AssuranceLevelMedium = "medium"
+
+func Test_CreateTargetOfEvaluation(t *testing.T) {
+	type fields struct {
+		cloudServiceHooks     []orchestrator.CloudServiceHookFunc
+		results               map[string]*assessment.AssessmentResult
+		AssessmentResultHooks []func(result *assessment.AssessmentResult, err error)
+		storage               persistence.Storage
+		metricsFile           string
+		loadMetricsFunc       func() ([]*assessment.Metric, error)
+		catalogsFile          string
+		events                chan *orchestrator.MetricChangeEvent
+	}
+	type args struct {
+		ctx context.Context
+		req *orchestrator.CreateTargetOfEvaluationRequest
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    assert.ValueAssertionFunc
+		wantErr bool
+	}{
+		{
+			name: "valid",
+			fields: fields{
+				storage: testutil.NewInMemoryStorage(t, func(s persistence.Storage) {
+					err := s.Create(&orchestrator.CloudService{Id: "MyService"})
+					assert.NoError(t, err)
+
+					err = s.Create(orchestratortest.NewCatalog())
+					assert.NoError(t, err)
+				}),
+			},
+			args: args{req: &orchestrator.CreateTargetOfEvaluationRequest{
+				Toe: orchestratortest.NewTargetOfEvaluation(),
+			}},
+			want: func(tt assert.TestingT, i1 interface{}, i2 ...interface{}) bool {
+				svc := i2[0].(*Service)
+
+				// We want to assert that certain things happened in our database
+				var toes []*orchestrator.TargetOfEvaluation
+				err := svc.storage.List(&toes, "", false, 0, -1, gorm.WithoutPreload())
+				if !assert.NoError(t, err) {
+					return false
+				}
+				if !assert.Equal(t, 1, len(toes)) {
+					return false
+				}
+
+				var service orchestrator.CloudService
+				err = svc.storage.Get(&service, "id = ?", "MyService")
+				if !assert.NoError(t, err) {
+					return false
+				}
+
+				return assert.Equal(t, 1, len(service.CatalogsInScope))
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			svc := &Service{
+				cloudServiceHooks:     tt.fields.cloudServiceHooks,
+				results:               tt.fields.results,
+				AssessmentResultHooks: tt.fields.AssessmentResultHooks,
+				storage:               tt.fields.storage,
+				metricsFile:           tt.fields.metricsFile,
+				loadMetricsFunc:       tt.fields.loadMetricsFunc,
+				catalogsFile:          tt.fields.catalogsFile,
+				events:                tt.fields.events,
+			}
+
+			gotRes, err := svc.CreateTargetOfEvaluation(tt.args.ctx, tt.args.req)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("Service.CreateTargetOfEvaluation() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+
+			if tt.want != nil {
+				tt.want(t, gotRes, svc)
+			}
+		})
+	}
+}
+
+func Test_GetTargetOfEvaluation(t *testing.T) {
```

Comment:
```suggestion
func TestService_GetTargetOfEvaluation(t *testing.T) {
```

Tests should follow the syntax `TestMyGlobalFunc` if it is a global fund or `TestMyStruct_MyFunc` if it is a method function of `MyStruct`.